dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -97,79 +168,448 @@ export const DynamicPackage = ({
+					>
+						<FrontCard
+							trail={card}
+							containerPalette={containerPalette}
+							containerType="dynamic/package"
+							showAge={showAge}
+							supportingContent={card.supportingContent}
+						/>
+					</LI>
+				);
+			})}
+			{columns.map((columnCards) => {
+				return (
+					<LI
+						showDivider={true}
+						showTopMarginWhenStacked={true}
+						percentage="25%"
+					>
+						<UL direction="column">
+							{columnCards.map((card, cardIndex) => {
+								return (
+									<LI
+										key={card.url}
+										padSides={true}
+										showTopMarginWhenStacked={false}
+										padBottom={
+											// No bottom margin on the last card
+											cardIndex < cards.length - 1
+										}
+										padBottomOnMobile={false}
+									>
+										<FrontCard
+											trail={card}
+											containerPalette={containerPalette}
+											containerType="dynamic/package"
+											showAge={showAge}
+											supportingContent={
+												card.supportingContent
+											}
+											imageUrl={undefined}
+										/>
+									</LI>
+								);
+							})}
+						</UL>
+					</LI>
+				);
+			})}
+		</UL>
+	);
+};
+
+const Card25_ColumnOfTwo25_ColumnOfTwo25_ColumnOfTwo25 = ({
+	cards,
+	containerPalette,
+	showAge,
+}: {
+	cards: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+}) => {
+	const bigs = cards.slice(0, 1);
+	const columns = [cards.slice(1, 3), cards.slice(3, 5), cards.slice(5, 7)];
+
+	return (
+		<UL direction="row">
+			{bigs.map((card) => {
+				return (
+					<LI
+						key={card.url}
+						padSides={true}
+						showTopMarginWhenStacked={false}
+						padBottom={false}
+						padBottomOnMobile={true}
+						percentage="25%"
+						showDivider={true}
+					>
+						<FrontCard
+							trail={card}
+							containerPalette={containerPalette}
+							containerType="dynamic/package"
+							showAge={showAge}
+							supportingContent={card.supportingContent}
+						/>
+					</LI>
+				);
+			})}
+			{columns.map((columnCards) => {
+				return (
+					<LI
+						showDivider={true}
+						showTopMarginWhenStacked={true}
+						percentage="25%"
+					>
+						<UL direction="column">
+							{columnCards.map((card, cardIndex) => {
+								return (
+									<LI
+										key={card.url}
+										padSides={true}
+										showTopMarginWhenStacked={false}
+										padBottom={
+											// No bottom margin on the last card
+											cardIndex < cards.length - 1
+										}
+										padBottomOnMobile={false}
+									>
+										<FrontCard
+											trail={card}
+											containerPalette={containerPalette}
+											containerType="dynamic/package"
+											showAge={showAge}
+											supportingContent={
+												card.supportingContent
+											}
+											imageUrl={undefined}
+										/>
+									</LI>
+								);
+							})}
+						</UL>
+					</LI>
+				);
+			})}
+		</UL>
+	);
+};
+
+const Card75_ColumnOfX25 = ({
```

Comment:
```suggestion
const Card75_ColumnOfCards25 = ({
```

What do you think? 'X' is perhaps a clearer indication of 'any number' but somehow it's maybe nice to keep things wordy